decidim_app-design/Gemfile.lock

```diff
@@ -47,10 +46,6 @@ PATH
     decidim-comments (0.11.0.pre)
       decidim-core (= 0.11.0.pre)
       jquery-rails (~> 4.0)
-    decidim-consultations (0.11.0.pre)
```

Comment:
Why this?

Reply:
No idea. Sometimes when I merge from `master`, I got this. I presume I do (or I don't do) something wrong, in any case, it's unintencioned.